client/src/templates/Challenges/components/monaco-editor.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  SandpackStack,
+  useActiveCode,
+  useSandpack
+} from '@codesandbox/sandpack-react';
+import Editor from '@monaco-editor/react';
+import React from 'react';
+
+const MonacoEditor = () => {
+  const { code, updateCode } = useActiveCode();
+  const { sandpack } = useSandpack();
+
+  return (
+    <SandpackStack>
+      <Editor
+        language={sandpack.activeFile.split('.').pop()}
+        theme='vs-dark'
+        key={sandpack.activeFile}
+        defaultValue={code}
+        onChange={value => updateCode(value || '')}
+        options={{
+          minimap: { enabled: false },
+          lineNumbers: 'on'
```

Comment:
Something is going wrong with the styling. The number of the current line is clearly visible, but the others blend into the background:

<img width="69" height="72" alt="Image" src="https://github.com/user-attachments/assets/a8e0f430-c447-43fc-b4cd-70e12dcf2b9a" />